starter-code/main.js

```diff
@@ -0,0 +1,284 @@
+						db.collection('companies').find({name: "Facebook"}).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "9":
+						db.collection('companies').find({name: "Facebook"}, {number_of_employees: 1, _id: 0}).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "10":
+						db.collection('companies').find({name: "Facebook"}, {"products.name": 1, _id: 0}).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "11":
+						db.collection('companies').find(
+								{name: "Facebook", "relationships.is_past": false},
+								{"relationships": 1, _id: 0}
+							).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								// console.log(JSON.stringify(result));
+								// This is a correction
+								result[0].relationships.filter((person) => {
+									return person.is_past === false;
+								}).forEach((worker) => {
+									console.log(`${worker.person.first_name} ${worker.person.last_name}`);
+								});
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "12":
+						db.collection('companies').find({name: "Facebook", "relationships.is_past": true}).count((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "13":
+						db.collection('companies').find({"relationships.person.permalink": "david-ebersman"}, {name: 1, _id:0}).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "14":
+						db.collection('companies').find({"competitions.competitor.name": "Facebook"}, {name: 1, _id: 0}).toArray((error, result) => {
```

Comment:
You can search the competitor of Facebook instead of the companies that have Facebook as competitor:
```
db.collection('companies').find({name: "Facebook"}, {"competitions": 1}) //...
```